dlp/src/test/java/dlp/snippets/RedactTests.java

```diff
@@ -62,10 +67,14 @@ public void tearDown() {
   }
 
   @Test
-  public void testRedactImage() {
+  public void testRedactImage() throws IOException {
     RedactImageFile.redactImageFile(PROJECT_ID, "src/test/resources/test.png");
 
     String output = bout.toString();
     assertThat(output, containsString("Redacted image written"));
+
+    // Clean up test output
+    Path outputFile = Paths.get("redacted.png");
```

Comment:
This should be in a separate cleanup function with the `After` tag, so it still runs of the test fails.

Reply:
moved this to a cleanup function in 65be2be